### ISSUE TITLE
PT Request a Change - Partner Information 

### ISSUE THREAD

**Issue by HRAGANBC** [bcgov/SIMS#3128]

**Comment by HRAGANBC** [comment]
@Joshua-Lakusta @CarlyCotton I believe this should be the last PT request a change form needed?

If we mimic the questions verbatim as they appear on the 'self-report' application section, it includes a mandatory tick-box indicating student has their partner's consent to report these changes. Do you think appropriate to include during Request a Change form, or should we rely exclusively on the Declaration which will be included for every Request a Change once #2555 is implemented. I.e. student would re-sign their entire declaration, which includes them taking responsibility for their partners current and future application information.

**Comment by michesmith** [comment]
add marital status questions and make it mandatory that they submit supporting docs  (given no verification with CRA) . Need an explanation.
- add dynamic validation (if they select single/ divorced) they don't need to provide additional marital details.

**Comment by andrepestana-aot** [comment]
@CarlyCotton @Joshua-Lakusta @HRAGANBC  We have some questions about this part of the workflow below: 
<img width="1170" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBNlJQQlE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--4501049c38ed0ebd1bc58f7601b69df93fbd01ad/image.png" alt="image.png" />

1. We noticed that we have a condition that checks if the student is independant and we'd like to confirm that this check is still required;
2. We only have a path in the worklfow for:
  a. (studentDataRelationshipStatus = "single" or studentDataRelationshipStatus = "other") and studentDataDependantstatus = "independant"
  b. studentDataRelationshipStatus = "married"
We don't have a path for a student that is single and dependant for example.
3. If that is still required, should we recalculate `studentDataDependantstatus` based on the data from the appeal?
4. There's a `calculatedDataPartnerStudentStudyWeeks` but it's not being used anywhere. Should we just remove it?

**Comment by CarlyCotton** [comment]
@andrepestana-aot 

1. There is no independent/dependent consideration in part-time. We only need to know if they are **married/common-law** or **NOT**.
2. We only need married/common-law (for family size and supporting partner income information purposes)
3. Not required beyond married or single
4. we can remove it in part-time, it's not used. It hasn't been included in any prior tickets for removal as it's not receiving any form input and not changing anything in the calculation.

**Comment by andrepestana-aot** [comment]
@CarlyCotton @HRAGANBC @Joshua-Lakusta 
I couldn't find where the values for these questions are used in the workflow for Part-time:
<img width="842" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBNFZWQlE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--e45b5291885f8d3926a1425f6f77a0399a6d7512/image.png" alt="image.png" />

Are those questions just used to support the ministry decision?

**Comment by HRAGANBC** [comment]
@andrepestana-aot 

Correct, the above referenced questions should not have any workflow impact. That is expected. Those questions indeed are there to support troubleshooting with the student. 

@CarlyCotton FYI - I believe these were the ones we decided against using because using them would mean 'double counting' the summary income field reported earlier (Line 15000)

**Comment by CarlyCotton** [comment]
@HRAGANBC That's right, these ones are NOT impacting the assessment because they should all be included in the value in the initial total income field. They are also being **_removed_** in ticket #3342.

@andrepestana-aot FYI

**Comment by HRAGANBC** [comment]
@andrepestana-aot 

Apologies for the back and forth, but after Carly pointed out that those questions are being removed from the application form itself (in #3342), I think it makes sense to remove them from the Request a Change form as part of this ticket, as well. I.e. the application form should match the Request a Change form. 

I will update AC's accordingly
